starport/interface/cli/starport/cmd/network_chain_join.go

```diff
@@ -34,14 +43,15 @@ func NewNetworkChainJoin() *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 	}
 	c.Flags().AddFlagSet(flagSetHomes())
+	c.Flags().String(flagGentx, "", fmt.Sprintf("Path to a gentx file (optional, requires --%s to be provided)", flagPeer))
+	c.Flags().String(flagPeer, "", fmt.Sprintf("Configure peer in node-id@host:port format (optional, requires --%s to be provided)", flagGentx))
 	c.Flags().String(flagKeyringBackend, "os", "Keyring backend used for the blockchain account")
 	return c
 }
 
 func networkChainJoinHandler(cmd *cobra.Command, args []string) error {
-	var (
-		chainID = args[0]
-	)
+	chainID := args[0]
+	gentxPath, _ := cmd.Flags().GetString(flagGentx)
```

Comment:
There are two types of errors.
Some errors cannot be determined during time of code is written because they're unkown at that period. E.g. i/o errors, invalid data.
Other ones can be known during time code is being written. If these errors break the program in the runtime then that can be considered as a programmer mistake (a bug).

In this one the error will always be nil until programmer mistakenly call a func like `GetInt` or use a wrong flag name.
I don't think we should handle errors that can be caused by programmers. Cobra pkg won't change this behavior in a minor release so, it should be safe.